src/packaging/metadata.py

```diff
@@ -20,10 +21,9 @@
 
 T = typing.TypeVar("T")
 
-
-try:
-    ExceptionGroup
-except NameError:  # pragma: no cover
+if "ExceptionGroup" in builtins.__dict__:  # pragma: no cover
+    ExceptionGroup = ExceptionGroup
```

Comment:
I don’t think this branch is needed? If ExceptionGroup is available, it’ll already be in the global scope.